testcases/pkcs11/sess_mgmt.c

```diff
@@ -1197,6 +1197,170 @@ CK_RV do_OperationState3(void)
     return rc;
 }
 
+CK_RV do_SessionCancel(void)
```

Comment:
The C_SessionCancel() function can still return with "not implemented", depending of the capabilities of the token, right? In these cases, the test should be skipped.

Reply:
In theory yes, but all our tokens support it since its implemented in common code.
Your statement would be true for almost all C_xxx functions of PKCS#11, so we should add it all over the place. However, this are opencryptoki testcases, so we can assume to know what opencryptoki implements.

Reply:
ok